apps/server/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jellyfish-server",
-  "version": "0.0.1",
+  "version": "0.0.4",
```

Comment:
Is this change needed?

Reply:
Not really, I used while testing locally, so rolling it back